### ISSUE TITLE
search box on the search page provides a false affordance

### ISSUE THREAD

**Issue by IanMulvany** [peerlibrary/peerlibrary#470]
in Safari when I mouse over the search box on https://peerlibrary.org/s I get a flashing prompt where the search text should be entered, but keyboard presses are not registered so I can't type any search text until I click on the text area.
